fi.json

```diff
@@ -280,7 +280,7 @@
 			"option-catalyst-desc": "Sinulla on tällä hetkellä {{tier}} Catalyst-lisenssi. Kiitos tuestasi!",
 			"option-catalyst-desc-no-license": "Sinulla ei tällä hetkellä ole Catalyst-lisenssiä. Catalyst-lisenssi on kertalahjoitus Obsidianin kehityksen tukemiseen.",
 			"label-app": "Sovellus",
-			"label-add-own-language": "See how to add your language to Obsidian."
+			"label-add-own-language": "Katso kuinka voit lisätä oman kielesi Obsidianiin."
```

Comment:
Please don't translate the word "Obsidian", it's a brand name and should never be translated.

Reply:
Hey, would Obsidian-sovellus be alright? It translates to Obsidian-application.

Having the unmodified word doesn't really sit in the sentence.

Also noticed the same thing happening on line 296 on the existing translations.

Reply:
Pushed the suggestions for review